mozetl/taar/taar_similarity.py

```diff
@@ -11,61 +11,66 @@
 import click
 import json
 import logging
+import numpy
 
 from pyspark.sql import SparkSession
 from pyspark.ml.feature import HashingTF, IDF
 from pyspark.ml.clustering import BisectingKMeans
 from pyspark.ml import Pipeline
+from pyspark.mllib.stat import KernelDensity
+from scipy.spatial import distance
 from taar_utils import store_json_to_s3
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 RAND_SEED = 42
+KDE_KERNEL_BANDWIDTH = 0.35
 
 
 def get_samples(spark):
     """ Get a dataframe with a valid set of sample to base the next
     processing on.
     """
-    return spark.sql("SELECT * FROM longitudinal")\
-        .where("active_addons IS NOT null")\
-        .where("size(active_addons[0]) > 2")\
-        .where("size(active_addons[0]) < 100")\
-        .where("normalized_channel = 'release'")\
+    return spark.sql("SELECT * FROM longitudinal") \
```

Comment:
Please drop all the whitespace changes to this file, as they would only complicate the rebasing.